spec/std/exception_spec.cr

```diff
@@ -0,0 +1,27 @@
+require "spec"
+
+# See https://github.com/crystal-lang/crystal/pull/3194#issuecomment-242474441
+class Exception
+  def inspect_with_backtrace(io : IO)
+    io << message << " (" << self.class << ")\n"
+    backtrace?.try &.each do |frame|
```

Comment:
Wouldn't be better this fix is part of the code itself (`src/exception.cr`) and not just the specs? /cc @bcardiff